File: la_rive_admin/ClientApp/src/app/item/item.ts
import { Category } from "../models/category";
import { Order } from "../models/order";

export class Item {

  public Item() {

  }

  id: string;
  itemName: string;
  itemNr: string;
  payAmount: number;
  sellAmount: number;
  count: number;
  availableCount: number;
  imgData: any;
  size: string;
  category: Category;
  order: Order;
}

File: la_rive_admin/Models/Item.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace la_rive_admin.Models
{
    [Table("Item")]
    public class Item
    {
        public Item()
        {
        }

        public string Id { get; set; }

        public string ItemNr { get; set; }
        public string ItemName { get; set; }
        public double PayAmount { get; set; }
        public double SellAmount { get; set; }
        public int Count { get; set; }
        public int AvailableCount { get; set; }
        public string ImgData { get; set; }
        public string Size { get; set; }

        public string CategoryId { get; set; }
        public string OrderId { get; set; }

    }
}
